File: sample/sample/SampleKitLoader.m
//
//  SampleKitLoader.m
//  sample
//
//  Created by sangmin park on 12/26/24.
//

#import <Foundation/Foundation.h>
#import "SampleKitLoader.h"
#import "Sample/sample-Swift.h"


@implementation SampleKitLoader
-(id<ModuleBase>) mount{
    id<ModuleBase> module = [[SampleKit alloc] init];
    return module;
}
@end

File: sample/sample/SampleKitLoader.swift
//
//  SampleKitLoader.swift
//  sample
//
//  Created by sangmin park on 11/28/24.
//

import Foundation
import MiniPubSub

@objc public class SampleKitLoader : ModuleLoader {
    override public func load() -> ModuleBase? {
        return SampleKit()
    }
}
